### PR TITLE
[SUC-304] updated titles to toggle if class is visible

### PR DESCRIPTION
**Pull request by SamaneriVimala**
Changed title attribute to toggle on and off dependent on whether Textual Information is on or off.
Other titles are on all the time if the css shows that the specific class is visible when TI is off.

### CODE REVIEW

File: client/elements/text/sc-simple-text.html [hunk 1]
```diff
@@ -278,6 +288,10 @@ <h3>Could not find the sutta text.</h3>
                       const classrefs = this.$.simple_text_content.querySelectorAll('.'+key);
                       Array.from(classrefs).forEach(item => item.title = this.classTitles[key]);
                   }
+                  for (var key in this.classTitlesTI){
+                      const classrefsTI = this.$.simple_text_content.querySelectorAll('.'+key);
+                      visible ? Array.from(classrefsTI).forEach(item => item.title = this.classTitlesTI[key]) : Array.from(classrefsTI).forEach(item => item.title = "");
```
Comment (gradam) [inline client/elements/text/sc-simple-text.html]: Line to long

File: client/elements/text/sc-simple-text.html [hunk 2]
```diff
@@ -278,6 +288,10 @@ <h3>Could not find the sutta text.</h3>
                       const classrefs = this.$.simple_text_content.querySelectorAll('.'+key);
                       Array.from(classrefs).forEach(item => item.title = this.classTitles[key]);
                   }
+                  for (var key in this.classTitlesTI){
+                      const classrefsTI = this.$.simple_text_content.querySelectorAll('.'+key);
```
Comment (gradam) [inline client/elements/text/sc-simple-text.html]: Please use string literals

File: client/elements/text/sc-simple-text.html [hunk 3]
```diff
@@ -133,18 +133,28 @@ <h3>Could not find the sutta text.</h3>
                       statePath: 'textOptions.chineseLookupTargetDictRepr',
                       observer: '_chineseLookupTargetDictReprChanged'
                   },
-                  classTitles: {
+                  classTitlesTI: {
```
Comment (TrebuhD) [inline client/elements/text/sc-simple-text.html]: I'd say don't worry about long names, it matters most if they're descriptive. I'd suggest naming this `textualInfoClassTitles` or `textualInfoClassDescriptions`.

File: client/elements/text/sc-simple-text.html (hunk 1 above)
Comment (TrebuhD) [inline client/elements/text/sc-simple-text.html]: I'd advise keeping complicated logic out of ternary operators (`? :`) - it's just harder to read. I'd rewrite this as:
```
if (visible) {
    Array.from(classrefsTI).forEach(item => item.title = this.classTitlesTI[key])
} else {
    Array.from(classrefsTI).forEach(item => item.title = "");
}
```

[Here's](https://github.com/airbnb/javascript#comparison--nested-ternaries) the relevant part of the Airbnb JS style guide.

Comment (SamaneriVimala) [reply client/elements/text/sc-simple-text.html]: OK. Good. Will change that. I actually had that at first but changed it.

File: client/elements/text/sc-simple-text.html (hunk 3 above)
Comment (SamaneriVimala) [reply client/elements/text/sc-simple-text.html]: OK. Will change that.

File: client/elements/text/sc-simple-text.html (hunk 1 above)
Comment (gradam) [reply client/elements/text/sc-simple-text.html]: Please use single quotes instead of double quotes.

**Comment by SamaneriVimala** [comment]
I've made the changes and pushed them to the branch. Now what?

**Pull request merged**